ui/nuxeo-path-suggestion/nuxeo-path-suggestion.js

```diff
@@ -231,6 +212,9 @@ import { FormatBehavior } from '../nuxeo-format-behavior.js';
               }
             });
         }
+        if (this.autoValidate) {
```

Comment:
if you're removing auto-validate above, what is this autoValidate in here? is not listed as a property of the component either so I am confused as where it's coming from

Reply:
Not removing, just not using the paper-typeahead to trigger the validation, but the nuxeo-path-suggestion instead (see the other comment).